templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -6,19 +6,9 @@
         <p>{{ 'error.intro'|trans({'status_code': status_code, 'status_text': status_text}, 'w3c_website_templates_bundle') }}</p>
     </div>
 
-    {% block stacktrace %}
-        {% if is_granted('ROLE_ADMIN') and exception %}
-            <div class="component component--text">
-                <h2>Error details</h2>
-                <p>Message: <strong>{{ exception.message }}</strong></p>
-                <p>in <em>{{ exception.file }}: {{ exception.line }}</em></p>
-                <p>Trace:</p>
-                <pre>{{ exception.traceAsString }}</pre>
-            </div>
-        {% endif %}
-    {% endblock stacktrace %}
-
     {% block content %}{% endblock content %}
+
+    {% block contact %}
     <p>
```

Comment:
shouldn't this be a text component? `<div class="component component--text">`

Reply:
indeed. Done in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/ff244a26f8f64f30e0d793b345a4a418b5c14082